test/integration/setup.ts

```diff
@@ -16,10 +16,7 @@ export async function suiteClientSetup(this: Mocha.Context): Promise<void> {
   // two times to give time to server to send `ledgerClosed` event
   // so getLedgerVersion will return right value
   await ledgerAccept(this.client)
-  const response = await this.client.request({
-    command: 'ledger',
-    ledger_index: 'validated',
-  })
+  const response = await this.client.getLedgerIndex()
   const ledgerVersion = response.result.ledger_index
   this.startLedgerVersion = ledgerVersion
```

Comment:
Looks like this part still needs to be consolidated?

Reply:
yep, that was an error, already fixed